services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/utils/GoalStateManager.java

```diff
@@ -198,6 +201,17 @@ public boolean shouldSkipClass(Class<?> clazz) {
                                   .setSubnetId(fixedIp.getSubnetId())
                                   .build();
                           fixedIps.add(fixedIp1);
+                            DHCP.DHCPConfiguration dhcpConfiguration=DHCP.DHCPConfiguration.newBuilder()
+                                    .setRevisionNumber(1)
```

Comment:
Let us use constant variables and collect them in one place so that we could easily change the value if needed.

Reply:
Ping for this comment @haboy52581